internal/bridge/stdout_scanner.go

```
@@ -0,0 +1,149 @@
+		// we somehow exceed the buffer — acceptable per spec.
+		select {
+		case s.errors <- StdoutError{Pattern: p.label, Line: truncated}:
+		default:
+		}
+	}
+}
+
+// matchesAnyErrorPattern reports whether line matches any of the scanner
+// patterns. Used from tests that want to verify the pattern set without
+// running the full pump goroutine.
+func matchesAnyErrorPattern(line string) (label string, ok bool) {
```

golangci-lint (2.11.4)

[error] func matchesAnyErrorPattern is unused

(unused)